### DISCUSSION TITLE
Problem connecting to MSSQL instance

### DISCUSSION THREAD

**Discussion by mistigtncc** [airbytehq/airbyte#33295] [Connector Questions]
Hey everyone. I am encountering a problem trying to use the SQL Server connector for my DB.
I have a SQL Server 2019 with a SQL instance name. Where should I specify this instance name? This is what I tried without success:

- In the HOST: my_ip\instance_name
- In the HOST: my_ip\\instance_name
- In JDBC URL param: instance=instance_name
- In JDBC URL param: instanceName=instance_name

I am still unable to connect.
I put encrytped to true with trust server sertificate.
I have no problem connection to my DB with python SQL Alchemy, so the ports are open.

Thank you !

**Comment by marcosmarxm** [comment]
Is this for a source or destination MSSQL?

**Reply by mistigtncc** [reply]
It is a source. I have an error while testing ( timeout )
